nion/swift/model/DocumentModel.py

```diff
@@ -2471,8 +2471,12 @@ def _get_builtin_processing_descriptions(cls) -> typing.Dict[str, typing.Any]:
                 "sources": [{"name": "src", "label": _("Source"), "croppable": True}], "parameters": [do_transpose_param, do_flip_v_param, do_flip_h_param]}
             width_param = {"name": "width", "label": _("Width"), "type": "integral", "value": 256, "value_default": 256, "value_min": 1}
             height_param = {"name": "height", "label": _("Height"), "type": "integral", "value": 256, "value_default": 256, "value_min": 1}
-            vs["rebin"] = {"title": _("Rebin"), "expression": "xd.rebin_image({src}.cropped_display_xdata, (height, width))",
```

Comment:
Similar to my comment in `niondata`, can you _add_ a new function `rebin_factor` and leave the old one in place for backwards compatibility?